qtawesome/tests/test_qtawesome.py

```diff
@@ -0,0 +1,14 @@
+r"""
+Tests for QtAwesome
+"""
+# Standard library imports
+import subprocess
+
+# Test Library imports
+import pytest
+
+def test_segfault_import():
+    subprocess.call('python -c qtawesome')
```

Comment:
Please this to

    output_number = subprocess.call('python -c qtawesome')
    assert  output_number == 0

Remember that you always have to `assert` something on tests :-)